utils/releaser/src/ReleaseWorker/ReleaseCandidate/UpdateUpgradeReleaseWorker.php

```diff
@@ -58,15 +66,24 @@ public function work(
             - check the correctness of the order of Shopsys packages and sections,
             - check whether there are no duplicated instructions for modifying docker related files,
             - check links whether they point to the repository in the "%s" version
-            - make sure, that every subsection of UPGRADE notes has link to correct pull request
-            - check "see #project-base-diff to update your project" is added to the all the upgrade notes and add the line where it is missing (at least to all the Storefront entries)
-            - replace all occurrences of #project-base-diff with link to project-base commit of the change
-                - you can find the commit hashes quickly by executing following on the project-base repository:
-                  git log --oneline --format="%%H %%s" | grep -E \'\(#[0-9]+\)$\'',
+            - make sure, that every subsection of UPGRADE notes has link to correct pull request',
             $versionString,
         ));
 
         $this->confirm('Confirm that all subsections of UPGRADE notes has their links to correct pull request.');
+
+        $this->symfonyStyle->note(sprintf(
+            '
+Now will run process that automatically replaces #project-base-diff hashes with links.
+During the process you might be asked to fill correct commit SHA1 as it might not be automatically retrieved.
+This command (that needs to be run in project-base repository with %s branch) might help you in looking for correct commit:
+git log --oneline --format="%%H %%s" | grep "COMMIT_MESSAGE_OF_MERGE_COMMIT_FROM_PR"',
```

Comment:
It seems to me more understandable this way :slightly_smiling_face: 
```suggestion
git log --oneline --format="%%H %%s" | grep "<put_here_commit_message_of_merge_commit_from_pr>"',
```